examples/UKCore-Bundle-Batch-Example.xml

```diff
@@ -0,0 +1,18 @@
+<Bundle xmlns="http://hl7.org/fhir">
+ <id value="UKCore-Bundle-Batch-Example" />
+ <type value="batch" />
+ <entry>
+  <request>
+   <method value="GET" />
+   <url value="https://fhir.hl7.org.uk/Patient/example" />
+  </request>
+   <request>
```

Comment:
fix needs to be in irts own entry tag

Reply:
why can't we have multiple requests in a single entry as that's how it has been done in json and other example which i saw?
{
   "type" : "batch",
   "resourceType" : "Bundle",
   "entry" : [
      {
         "request" : {
            "method" : "GET",
            "url" : "CodeSystem/$lookup?system=http://loinc.org&code=23245-4"
         }
      },
      {
         "request" : {
            "method" : "GET",
            "url" : "CodeSystem/$lookup?system=http://snomed.info/sct&code=17311000168105"
         }
      }
   ]
}

Reply:
i have done the change still I don't know FHIR Validation call is failing